js/components/bookmarksToolbar.js

```diff
@@ -231,12 +257,15 @@ class BookmarksToolbar extends ImmutableComponent {
     contextMenus.onTabsToolbarContextMenu(this.props.activeFrame, closest && closest.props.bookmark || undefined, closest.isDroppedOn, e)
   }
   render () {
+    let showFavicon = getSetting(settings.SHOW_BOOKMARKS_TOOLBAR_FAVICON) === true
     this.bookmarkRefs = []
     return <div
-      className={cx({
-        bookmarksToolbar: true,
-        allowDragging: this.props.shouldAllowWindowDrag
-      })}
+      className={
+        cx({
+          bookmarksToolbar: true,
+          allowDragging: this.props.shouldAllowWindowDrag
+        }) + (showFavicon ? ' showFavicons' : '')
```

Comment:
I'll merge this and fix but just an fyi that you can do this:
```
 cx({
  bookmarksToolbar: true,
  allowDragging: this.props.shouldAllowWindowDrag,
  showFavicons: showFavicon
})
```

It's basically just a way to have object properties with true/false values and if they are true the key name is converted to a string.  It's a nicer format than a bunch of ternary checks because it's so common to include / exclude for classes.